doc/code_examples/Tutorial_Final.cpp

```diff
@@ -182,17 +180,9 @@ class TOPPDatabaseFilter :
      vector<ProteinIdentification> protein_identifications;
      vector<PeptideIdentification> peptide_identifications;
 
-     if (ids_type == FileTypes::IDXML)
+     if (!FileHandler().loadIdentifications(ids, protein_identifications, peptide_identifications, {FileTypes::IDXML, FileTypes::MZIDENTML}))
```

Comment:
hmm what is the semantics of the return value == false?
- unknown input file type?
- missing input file?

Reply:
It means we don't have a handler for that type of file. If the file can't be created (for store) or can't be read (for load) we get the corresponding exceptions.

Reply:
ah got it.. must be at least an exception. Return value could go unnoticed.